lib/components/IndexContent.jsx

```diff
@@ -21,17 +32,16 @@ export const IndexContent = (
   const [network, setNetwork] = useState('kovan')
   const [contractAddress, setContractAddress] = useState('')
 
-  const kovanDaiPrizePoolContractAddress = getDemoPoolContractAddress('kovan', 'dai')
-  const kovanUsdcPrizePoolContractAddress = getDemoPoolContractAddress('kovan', 'usdc')
-  const kovanUsdtPrizePoolContractAddress = getDemoPoolContractAddress('kovan', 'usdt')
+  const demoNetworkName = _.findKey(demoPools, {chainId: walletNetwork})
```

Comment:
I would typically do `import { findKey } from 'lodash'` prior to using it. I'm not even sure how lodash is working without being imported first?

Reply:
good catch; the symbol "_" is exported globally by lodash.  I am used to this style, but I will switch to explicit imports.

Reply:
fixed!